image-streams/imagesignature.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ImageSignature
+metadata:
+ name: sha256:29f5d56d12684887bdfa50dcd29fc31eea4aaf4ad3bec43daf19026a7ce69912@imagesignaturetest2
```

Comment:
How do you get above info? Do you mean you could not add <imagename>@<imagesignaturename> in this file? 
 I mean you use "name: <imagename>@<imagesignaturename>" ,and after you download this file,and replace the imagename and imagesignaturename using actual name.